examples_utils/notebook_logging/gc_logger.py

```diff
@@ -461,13 +468,21 @@ def __sanitize_payload(cls, payload: dict) -> dict:
         if cls.LOG_STATE == "DISABLED":
             return
 
-        # Clean out any private keys, fix quotes
-        for key, val in payload.items():
-            if (val is not None) and (type(val) == str):
-                if key in ["error_trace", "cell_output", "code_executed"]:
-                    val = cls.__remove_hf_keys(val)
+        # Clean out any private keys, fix quotes, remove None or empty fields
+        for key, val in payload.copy().items():
+            if val is not None:
+                if type(val) == str:
```

Comment:
This might be a less brittle comparison?

```suggestion
                if isinstance(val, str):
```